quaint/Cargo.toml

```diff
@@ -114,7 +114,7 @@ version = ">1.4.0"
 [dependencies.mysql_async]
 git = "https://github.com/prisma/mysql_async"
 optional = true
-branch = "vendored-openssl"
+branch = "fix/mariadb-11-version-detection"
```

Comment:
TODO: revert once https://github.com/prisma/mysql_async/pull/4 is merged (and update mysql_async)